util/minetestmapper.py

```diff
@@ -202,8 +271,30 @@ def usage():
 
 # Load color information for the blocks.
 colors = {}
+colors_path = "colors.txt"
+
+os_name="linux"
+if (os.path.sep!="/"):
```

Comment:
I'm sure there's a better vay to do this test.  `os` uses `"nt" in sys.builtin_module_names`.
Also `s/linux/posix/`.